aspnetcore/fundamentals/logging/index/samples/6.x/WebLog/Program.cs

```diff
@@ -1,58 +1,80 @@
-#define APP3 // FIRST SECOND APP3
+#define FIRST // FIRST SECOND APP3 APP4
 #if NEVER
 #elif FIRST
-#region snippet1
+// <snippet1>
 var builder = WebApplication.CreateBuilder(args);
 
-var logger = LoggerFactory.Create(config =>
-{
-    config.AddConsole();
-}).CreateLogger("Program");
+// Configure logging with AddConsole()
+builder.Logging.AddConsole();
 
 var app = builder.Build();
 
 app.MapGet("/", () => "Hello World!");
 
-app.MapGet("/Test", async context =>
+app.MapGet("/Test", async (ILogger logger, HttpResponse response) =>
 {
     logger.LogInformation("Testing logging in Program.cs");
-    await context.Response.WriteAsync("Testing");
+    await response.WriteAsync("Testing");
 });
 
 app.Run();
-#endregion
+// </snippet1>
 #elif SECOND
-#region snippet2
+// <snippet2>
 using Microsoft.Extensions.Logging.Console;
 
 var builder = WebApplication.CreateBuilder(args);
 
-using var loggerFactory = LoggerFactory.Create(builder =>
-{
-    builder.AddSimpleConsole(i => i.ColorBehavior = LoggerColorBehavior.Disabled);
-});
-
-var logger = loggerFactory.CreateLogger<Program>();
+// Configure logging with AddSimpleConsole
+builder.Logging.AddSimpleConsole(i => i.ColorBehavior = LoggerColorBehavior.Disabled);
 
 var app = builder.Build();
 
 app.MapGet("/", () => "Hello World!");
 
 app.MapGet("/Test", async context =>
```

Comment:
```suggestion
app.MapGet("/Test", async (ILogger<Program> logger, HttpResponse response) =>
{
    logger.LogInformation("Testing logging in Program.cs");
    await response.WriteAsync("Testing");
}
```
As written would also work, but given AddSimpleConsole() was the focus of this snippet I think we are better off keeping everything else consistent between snippet 1 and snippet 2.